nautobot/dcim/tests/test_filters.py

```diff
@@ -726,15 +751,20 @@ class SiteTestCase(FilterTestCases.NameSlugFilterTestCase, FilterTestCases.Tenan
     def setUpTestData(cls):
         common_test_data(cls)
 
+        cls.regions = Region.objects.filter(sites__isnull=True, parent__isnull=True, children__isnull=True)[:2]
```

Comment:
Should the factory test data generator explicitly create enough regions to match this criteria?